client/asset/dcr/dcr.go

```diff
@@ -1167,12 +1166,12 @@ func (dcr *ExchangeWallet) fund(enough func(sum uint64, size uint32, unspent *co
 
 // spendableUTXOs generates a slice of spendable *compositeUTXO.
 func (dcr *ExchangeWallet) spendableUTXOs() ([]*compositeUTXO, error) {
-	unspents, err := dcr.wallet.Unspents(dcr.ctx, dcr.acct)
+	unspents, err := dcr.wallet.Unspents(dcr.ctx, dcr.acctName)
 	if err != nil {
 		return nil, err
 	}
 	if len(unspents) == 0 {
-		return nil, fmt.Errorf("insufficient funds. 0 DCR available to spend in %q account", dcr.acct)
+		return nil, fmt.Errorf("insufficient funds. 0 DCR available to spend")
```

Comment:
If this error gets displayed to users, it would be helpful to include the fact that the unavailability of spendable dcr is specifically for a particular account, as users may have other accounts with spendable utxos. I think even if the error is only being logged, that additional info will be helpful.